packages/@azure/servicebus/data-plane/lib/core/messageReceiver.ts

```diff
@@ -761,7 +750,7 @@ export class MessageReceiver extends LinkEntity {
           _receiver: this._receiver
         };
         log.error(
-          "[%s] Something is busted. State of Receiver '%s' with address '%s' is: %O",
+          "[%s] Something went wrong. State of Receiver '%s' with address '%s' is: %O",
```

Comment:
Can we do a text scan in the entire repo and fix this error message everywhere?

Reply:
We have already fixed this error in Event Hubs. This error in left only in Service Bus. We are not using this text anywhere else.